test/test-applications/integrations/TestApplication.AspNet.NetFramework/Dockerfile

```
@@ -4,6 +4,11 @@ ARG windowscontainer_version=ltsc2022
 FROM mcr.microsoft.com/dotnet/framework/aspnet:4.8-windowsservercore-${windowscontainer_version}
 ARG configuration=Debug
 ARG platform=x64
+WORKDIR C:\opentelemetry
+COPY bin/tracer-home .
+WORKDIR C:\init
+COPY docker-init.ps1 .
+RUN [ "powershell", "-c", "C:\\init\\docker-init.ps1" ]
```

WORKDIR is set to `C:\init`
I think we can change this line to:

```suggestion
RUN [ "powershell", "-c", ".\\docker-init.ps1" ]
```